cpp/src/parquet/metadata.cc

```diff
@@ -58,6 +58,13 @@ const ApplicationVersion& ApplicationVersion::PARQUET_MR_FIXED_STATS_VERSION() {
   return version;
 }
 
+const ApplicationVersion& ApplicationVersion::PARQUET_CPP_10353_FIXED_VERSION() {
+  // Parquet 1.5.1 had this problem, and after that we switched to the
```

Comment:
Which problem? Neither the comment nor the function name explains it.

Reply:
The 10353 was meant to refer to the ARROW-10353 issue. I have added a summary of the issue (or specifically the part that makes use need to workaround) and referred explicitly to "ARROW-10353".